darwin/dataset/remote_dataset_v2.py

```diff
@@ -111,6 +111,7 @@ def push(
         *,
         blocking: bool = True,
         multi_threaded: bool = True,
+        max_workers: int = None,
```

Comment:
```suggestion
        max_workers: Optional[int] = None,
```

Reply:
All done.
@rslota hope you can resolve this one.